pkg/server/module_server.go

```diff
@@ -98,6 +102,11 @@ type ModuleServer struct {
 	buildBranch string
 
 	promGatherer prometheus.Gatherer
+	registerer   prometheus.Registerer
+
+	KVStore          kv.Config
```

Comment:
suggest using memberListKV or similar for this, since there is also an internal kvstore in grafana which might be confusing